torch/onnx/symbolic_opset9.py

```diff
@@ -851,7 +854,7 @@ def symbolic_fn(g, input, output_size):
         if mod != [0] * len(mod):
             if output_size == [1] * len(output_size):
                 return g.op("GlobalMaxPool", input), None
-            return _unimplemented(name, 'output size that are not factor of input size')
```

Comment:
Old behavior a warning is emitted.  It is changed to a runtime error?  there is one internal test that fails.  Is this something we want?

Reply:
Note other ops  all uses _unimplemented

Reply:
Before my update, this line in the symbolic conversion was throwing a warning instead of an error. Hence, the ATen node in the graph was not converted to ONNX, and was left as ATen.
By default, this behavior is not accepted. But exporter has a configuration mode that accepts combination of ONNX and ATen nodes. It's called ONNX_ATEN_FALLBACK.
It could be that the internal test case you see is failing is exporting with the ONNX_ATEN_FALLBACK mode.
I'll send a fix.